src/fusor/models.py

```diff
@@ -554,12 +555,13 @@ class Assay(BaseModelForbidExtra):
     )
 
 
-AssayedFusionElements = list[
+AssayedFusionElements = Annotated[
```

Comment:
I think we should change the name of this and the other Elements type to be singular, rather than plural, since it's no longer a list

Reply:
Similarly, I think `structure` should be renamed to `structures` since it is a list?

Reply:
should I change `structure` as part of this PR or in a new one? I originally renamed it to `structure` from `structural_elements` based on @jarbesfeld 's changes here: https://github.com/cancervariants/fusor/pull/151/files